<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Escala: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scaleMapCb"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="addRioBtn">
     <property name="text">
      <string>Rótulo de Rio</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="addLagoBtn">
     <property name="text">
      <string>Rótulo de Lago</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
